src/components/Sheep.tsx

```diff
@@ -110,8 +116,40 @@ export const Sheep = ({
   };
 
   const keyPressed = (p5: p5Types) => {
-    if (p5.keyIsDown(p5.UP_ARROW) || p5.key === "w") {
-      speed = -40;
+    switch (p5.key) {
+      case "a":
+        pressedKeys.a = true;
+        break;
+      case "d":
+        pressedKeys.d = true;
+        break;
+      case "ArrowLeft":
+        pressedKeys.left = true;
+        break;
+      case "ArrowRight":
+        pressedKeys.right = true;
+        break;
```

Comment:
This whole ``pressedKeys.Key`` looks redundant to me. I personally would much rather have something like this:
 ```ts
rightMoveList = ['d', 'ArrowRight']
leftMoveList = ['a', 'ArrowLeft']
```

and with that, dynamically match states e. g.:
```ts
if (rightMoveList.includes(p5.key)) 
  isRight = true
  
if (leftMoveList.includes(p5.key)) 
  isLeft= true
```
And so on.

I can't really explain why I feel like that would be an improvement, it just seems like less duplication.
Maybe you and others can share their opinion about it

Reply:
Yeah makes sense. Then it's also easier to add new keys for the controlling of the sheep in the future.

Reply:
@94726 This is a good idea. Setting `isRight` or `isLeft` in the `keyPressed` function causes problems when multiple keys are pressed at the same time, but I managed to simplify the code. @LeoTuet I also moved all keyboard shortcuts into a single object, making the addition of new key bindings easier.